source/manual/dns.html.md

```diff
@@ -15,76 +15,102 @@ GOV.UK is responsible for managing several DNS zones, spanning a number of `*.go
 
-We use a few domains:
+GOV.UK is responsible for more than just the `www.gov.uk` domain.
+The [GOV.UK proposition](https://www.gov.uk/government/publications/govuk-proposition/govuk-proposition#what-the-govuk-proposition-covers) lists the domains we're responsible for. These are:
 
-- `alphagov.co.uk` is the old domain name GOV.UK publishing used to live on.
-  We maintain records which point to Bouncer so that these URLs redirect.
-- `publishing.service.gov.uk` and `govuk.service.gov.uk` are where GOV.UK lives.
+- www.gov.uk
+- service.gov.uk
+- data.gov.uk
+- blog.gov.uk
+- campaign.gov.uk
+- independent.gov.uk
+- api.gov.uk
 
-## DNS for `*.service.gov.uk` domains
+There are lots of other `*.gov.uk` domains, such as `cityoflondon.gov.uk`, which are managed by Cabinet Office (who use Nominet).
 
-GOV.UK Technical 2nd Line are responsible for delegating DNS to other government services.
-Note that we __do not__ manage any other DNS records: if you get a request concerning anything other than `NS` records, it should be rejected.
+In theory, the GOV.UK proposition domains should all be managed by GOV.UK (who use [Jisc](https://www.jisc.ac.uk/): a non-profit that provides networking to UK education and government). In practice, a couple of GOV.UK proposition domains are managed by Cabinet Office, and a couple of non-proposition domains are in GOV.UK's Jisc account. [This is being looked at](https://trello.com/c/qNpyVaC5/3228-consolidate-co-vs-non-co-domains-in-govuks-jisc-account) by Platform Security & Reliability.
 
-When you've verified the authenticity of the request as per the SRE docs above, you should:
+In most cases, DNS zones are hosted by both AWS (Route 53) and Google Cloud Platform (Cloud DNS). See [Amazon Route53 vs Google Cloud in the govuk-dns-tf README](https://github.com/alphagov/govuk-dns-tf#amazon-route53-vs-google-cloud)
 
-1. Ensure you have [Terraform Cloud access](/manual/terraform-cloud.html)
-1. Commit your changes in [govuk-dns-tf][] (see [example](https://github.com/alphagov/govuk-dns-tf/pull/14))
-1. Push your changes to GitHub and open a pull request
-1. Terraform Cloud will automatically perform a plan. Open the [govuk-dns-tf][govuk-dns-tf-cloud] workspace to see it.
-1. If you are happy with the results of the plan, merge your PR
-1. From the PR page in GitHub, look under the pre-merge checks section and open the "details" link from the Terraform Cloud check.
-1. Press "Confirm and apply" in Terraform Cloud.
+### The `gov.uk` domain
 
-[govuk-dns-tf-cloud]: https://app.terraform.io/app/govuk/workspaces/govuk-dns-tf
+This is managed by Cabinet Office. `gov.uk.` is an apex domain so it [cannot have a CNAME record](https://tools.ietf.org/html/rfc1912#section-2.4). Instead, it has A records that point directly to Fastly virtual IP addresses, resolving to the [Production TLD Redirect](https://manage.fastly.com/configure/services/7IaQm6UK3NiQu0v0E83YKn) service, which performs a redirect to `www.gov.uk`.
```

Comment:
Yeah — and either way "The `gov.uk` domain ... is managed by Cabinet Office" is less than entirely accurate at best.

Reply:
Which bit of it are you not sure is true? "Managed by Cabinet Office", apex domain / A records, or production redirect? AFAICT that's the current state of play. I don't remember ever seeing a `gov.uk` domain in Jisc, suggesting that it moved to the CO account before I became a Lead.

I removed the bit saying how to make changes (via "emailing `naming@ja.net` from someone on Jisc's approved contacts") because I imagine that process _has_ changed - if gov.uk is managed by CO, then we're quite likely not on the list of approved contacts. It's also likely that only someone from Senior Tech would be actioning changes.

Reply:
@ChrisBAshton it seems to me you're conflating a bunch different aspects of "managing" the domain here. Probably worth a re-think of this doc in terms of: 1) who is the intended audience 2) what is the reader trying to do when they find this doc?

Then if you find it still makes sense to include this level of detail then we work together to make sure that what you're writing is accurate as well as meeting the reader's needs.

Reply:
e.g. much of this is unlikely to be relevant to GOV.UK developers and has potential to cause/amplify confusion rather than helping — more info isn't always better.

Reply:
I rephrased it to "administered by Cabinet Office", but perhaps "registered by Cabinet Office" makes more sense?
(Though I'm hoping the `www.gov.uk` domain will come under GOV.UK ownership and we can remove some of these surprises).

The intended audience is any engineer who wants to get an understanding of why we manage the domains we manage (the GOV.UK proposition otherwise has zero mentions in the Dev Docs so far), as well as the numerous gotchas for each of the said domains.

(The technicals around applying DNS changes are documented elsewhere - in govuk-dns-tf - and the policies are documented in the linked Google playbook).